training/tests/integration/config/benchmark.yaml

```diff
@@ -0,0 +1,32 @@
+# Modifications for the basic template "config.yaml"
+hardware:
+  files:
+  #TODO change to n320-o96 (transformer?)
```

Comment:
Yes we were discussing about this with @japols, maybe we could try to think for a different test where we just called the forward pass of the model with a matrix generated on the fly at n320/o1280?